install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -3047,6 +3047,12 @@ clustermesh:
       # @schema
       # -- The internalTrafficPolicy of service used for apiserver access.
       internalTrafficPolicy: Cluster
+      # -- Defines when to enable session affinity.
```

Comment:
It looks reasonable to define this option as an enum through the `@schema` comment, as done above.